src/lib/api-client.js

```diff
@@ -184,3 +202,14 @@ export function getModelsForNewItem() {
         hashHistory.push('/items/new');
     });
 }
+export function deleteItem(address){
+    return del('item', {
+        itemAddress: address
+    }).then(data => {
+        Dispatcher.handleAction('ITEMS_RECEIVED', data);
```

Comment:
The point of calling ITEMS_RECEIVED is b/c the store has changed so I make sure to pass that along. I didn't implement ITEM_DELETED because it would only be used for the toast which I was told to ignore and we can add a task when either toast-store is in DEV or this is merged.